sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,254 @@
+- Implemented `sycl::stream`'s size functions definition for device [368b6605]
+- Introduced an API to get thread ID and subdevice ID for ESIMD extension 
+[dd6145e8]
+- Implemented `kernel_bundle::native_specialization_constant()` [4a412307]
+- Implemented SYCL 2020 identities without a known or specified identity 
+[648845fb] [1404ff1d]
+- Implemented basic functionality for the following group types: `ballot_group`, 
+`cluster_group`, `tangle_group`, `opportunistic_group` as part of 
+[`sycl_ext_oneapi_non_uniform_groups`](doc/extensions/proposed/sycl_ext_oneapi_non_uniform_groups.asciidoc) extension [86894205]
+- Implemented `is_group` trait and changed the definition of `is_group_v` to be 
+in line with SYCL 2020 [10e07a38]
+- Implemented `atomic_memory_scope_capabilities` and 
+`atomic_memory_order_capabilities` device queries for OpenCL and Level Zero 
+backends [2816c14072f3] [b18e6ea83c62]
+- Implemented `atomic_fence_scope_capabilities` and 
+`atomic_fence_order_capabilities` device queries for OpenCL and Level Zero 
+backends [632946ac1cbf]
+- Implemented `sycl::ext::intel::experimental::esimd::bfn` function from ESIMD 
+extension [25d0475f]
+- Made `sycl::ext::oneapi::experimental::joint_reduce` work with 
+`sycl::sub_group` [cb91c232]
+
+### Documentation
+- Added [`sycl_ext_oneapi_annotated_ptr`](doc/extensions/experimental/sycl_ext_oneapi_annotated_ptr.asciidoc), [`sycl_ext_intel_fpga_kernel_arg_properties`](doc/extensions/proposed/sycl_ext_intel_fpga_kernel_arg_properties.asciidoc), and 
+[`sycl_ext_oneapi_kernel_arg_properties`](doc/extensions/proposed/sycl_ext_oneapi_kernel_arg_properties.asciidoc) extention specifications [b423e1db]
+- Reintroduced `device_ptr` and `host_ptr` under SYCL namespace for 
+[`sycl_ext_intel_usm_address_spaces`](doc/extensions/supported/sycl_ext_intel_usm_address_spaces.asciidoc) [ea8a0e56]
+- Moved [`sycl_ext_oneapi_memcpy2d`](doc/extensions/supported/sycl_ext_oneapi_memcpy2d.asciidoc) to supported [751acf35]
+- Updated documentation for `-fsycl-no-rdc` mode [a27e6fc5]
+- Added an initial draft of non-uniform sub-groups proposal [cc7cbd28] 
+[a5d62c8a]
+- Added a draft of 16-bit atomics extension [1b973aed]
+- Added an extension specification for USM allocation functions with properties 
+support [3600f922]
+- Added `sycl_ext_oneapi_peer_access` extention specification [b5bce772] 
+[37960cda]
+
+## Improvements
+
+### SYCL Compiler
+- Improved `sycl::reqd_work_group_size` for optional dimensions [b211d0df]
+- Enabled DPC++ build with llvm-mingw toolchain on Windows [1715c36b]
+- Deprecated usage of `-sycl-std=1.2.1` [8dd6fdff]
+- Disabled force inlining of kernel call operator for FGPA [90e0e650]
+- Reimplemented `-f[no]sycl-early-optimizations` flag [d164fd94]
+
+### SYCL Library
+- Started to use const qualifier in several accessor iteratoration operators 
```

Comment:
```suggestion
- Started to use const qualifier in several accessor iterator operators 
```

Tagging @maarquitos14. This commit seems to be a part of a series to fully enable accessor iterators, if I understand correctly. Perhaps we can omit listing it as a separate item

Reply:
Exactly, it was a fix required to make accessor iterators work in MSVC.